tests/test_adaptor_increase_maxorder.cc

```diff
@@ -183,7 +185,7 @@ namespace rascal {
 
         auto neighbour_position = pair.get_position();
         auto diff_pos_pair = (neighbour_position - atom_position).norm();
-        BOOST_CHECK_CLOSE(diff_pos_pair, 1., TOLERANCE);
+        BOOST_CHECK_CLOSE(diff_pos_pair, distances[counter++], TOLERANCE);
```

Comment:
I usually prefer to have separate statements for increasing the counter and indexing the array. This way, one do not have to remember the difference between `counter++` and `++counter`, and there are no copy/paste errors.

Feel free to disagree and leave the code as is though =)

Reply:
I have no strong feelings. Changed towards your proposition.